Library/Formula/relx.rb

```diff
@@ -0,0 +1,49 @@
+class Relx < Formula
+  desc "Sane, simple release creation for Erlang"
+  homepage "https://github.com/erlware/relx"
+  url "git@github.com:erlware/relx.git", :using => :git,
```

Comment:
You need to use the `https://github.com/erlware/relx.git` URL rather than the SSH version.